test/performance/odp_queue_perf.c

```diff
@@ -190,12 +190,12 @@ static int create_queues(test_global_t *global)
 		event[i] = ODP_EVENT_INVALID;
```

Comment:
There still seems to be quite a few error-like prints using `printf()`, swap those as well?

(E.g. 
```c
		printf("Too many queues %u. Test maximum %u.\n",
		       test_options->num_queue, MAX_QUEUES);

...
			printf("Max queues supported %u\n",
			       queue_capa.plain.max_num);
```
)

Reply:
Change in V2.